src/js/util/ui.js

```diff
@@ -114,8 +114,60 @@ define(function (require, exports) {
         }
     };
 
+    /**
+     * Parses the given refPoint string and returns the keys to be passed into
+     * a bounds object
+     *
+     * @param {string} refPoint Two character string denoting the active reference point
+     *                          [lmr][tcb]
+     *
```

Comment:
No newline is necessary here. Also, you don't need this much hanging indentation.